juju/juju
# [JUJU-2273] Add unused dry run option

This will be used when removing application to generate a preview of
what is going to be destroyed

Hard-code this option to false in the client for the time being

## Checklist


## QA steps

Verify unit tests run

With both new and old controllers, verify that `juju remove-application` functions as before
